declarations/str-utils/excercise11.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'str-utils' {
```

Comment:
The solution is correct although the rest of the code is missing in order to run it.